deploy/materialized_view_identifier.sql

```diff
@@ -0,0 +1,56 @@
+-- Deploy ggircs:materialized_view_identifier to pg
+-- requires: materialized_view_report
+
+begin;
+
+create materialized view ggircs_swrs.identifier as (
+  with reportXMLs as (
+    select _report.source_xml as source_xml,
+           _report.id         as report_id,
+           _report.swrs_report_id,
+           _report.imported_at,
+           _report.swrs_facility_id,
+           _facility.id       as facility_id
+    from ggircs_swrs.report as _report
+           inner join ggircs_swrs.facility as _facility on _report.id = _facility.report_id
+    order by _report.id asc
+  )
+  select row_number() over (order by report_id asc) as id,
+         report_id,
+         swrs_report_id,
+         facility_id,
+         swrs_facility_id,
+         identifier_type,
+         identifier_value,
+         -- imported_at,
+         row_number() over (
+           partition by swrs_facility_id, identifier_type
+           order by
+             report_id desc,
+             identifier_value desc
+           ) as swrs_identifier_history_id
+  from reportXMLs,
+       xmltable(
+           '/ReportData/RegistrationData/Facility/Identifiers/IdentifierList/Identifier/IdentifierValue[normalize-space(.)]'
+           passing reportXMLs.source_xml
+           columns
```

Comment:
maybe we can get the `swrs_facility_id` from a parent xml context rather than a join?